spec/spec_helper.rb

```diff
@@ -108,6 +108,7 @@ def clean_active_fedora_repository
   # The JS is executed in a different thread, so that other thread
   # may think the root path has already been created:
   ActiveFedora.fedora.connection.send(:init_base_path)
+  Hyrax.persister.wipe! if Hyrax.config.query_index_from_valkyrie
```

Comment:
i'm not following this. the `Hyrax.persister` is independent of the index, right?

Reply:
You would think that.  And perhaps this should be somewhere else, but I'm just using the existing `wipe!` method defined in wings persister.

https://github.com/samvera/hyrax/blob/4efa6b8f3133fbcb282cda7e791b5bbd7a07d3d7/lib/wings/valkyrie/persister.rb#L60-L65

Reply:
An alternative...

```
# define in Hyrax::SolrService
def wipe!
   Hyrax::SolrService.delete_by_query("*:*") 
   Hyrax::SolrService.commit 
end
```

```
# update in Wings::Valkyrie::Persister
def wipe!
  Hyrax::SolrService.wipe!
  ActiveFedora::Cleaner.clean!
end
```

Reply:
i don't think it's safe to assume that `Hyrax.persister` is a the wings persister (it's not for many tests).

i think i'm not understanding the use case in general, either. this method is named `#clean_active_fedora_repository`; when/why does that need to clear the custom, non-AF solr index?

Reply:
I added a `wipe!` method to `Hyrax::SolrService` and moved that call to the `:clean_repo` section of `spec_helper.rb` since that is what triggers the call from the spec.  I think the assumption here is that the writer of the test does want all repos (e.g. Fedora, Redis, Solr) cleared to start the tests with a clean slate, including Valkyrie solr if it is in use.  

Long term, this should perhaps call `Hyrax.persister.wipe!` with the theory that if `:clean_repo` is being called by a spec, then that persister was used.  Or we may want to make this more granular (e.g. `:wipe_hyrax_persister`, `:wipe_solr`, etc.).